packages/command-palette/src/use-command-palette.tsx

```diff
@@ -293,7 +318,18 @@ export const useCommandPalette = ( {
 
 		finalSortedCommands = finalSortedCommands.map( ( command: Command ) => {
 			const callback = ( params: CommandCallBackParams ) => {
-				const targetFunction = command?.siteFunctions?.onClick || command.callback;
+				let targetFunction;
+				const isMultiSiteCommand = command?.siteFunctions !== undefined;
+				if ( isMultiSiteCommand ) {
+					targetFunction = command?.siteFunctions?.onClick;
+					// We need to track the selected command event here because `command.siteFunctions.onClick`
+					// does not track anything (and it's not meant to).
+					trackSelectedCommand( command );
```

Comment:
This here needs to track the completed command. All this logical branch only gets executed when we have a selected current site. In that context what we're doing is mapping the multiSiteCommand to a singleSiteCommand and the only possible outcome is `trackCompletedCommand`.

Reply:
I think I solved this in https://github.com/Automattic/wp-calypso/pull/88285/commits/c91689a61ceb9638e378425472f8937737f26e8f